src/core/plugins/json-schema-5-samples/fn/index.js

```diff
@@ -635,3 +667,7 @@ const resolver = (arg1, arg2, arg3) => [arg1, JSON.stringify(arg2), JSON.stringi
 export const memoizedCreateXMLExample = memoizeN(createXMLExample, resolver)
 
 export const memoizedSampleFromSchema = memoizeN(sampleFromSchema, resolver)
+
+export const getSchemaObjectTypeLabel = (schema) => getType(immutableToJS(schema))
+
+export const getSchemaObjectType = (schema) => schema?.get("type") ?? "string"
```

Comment:
Not sure about it but maybe we should we be handling immutable here as well?

```suggestion
export const getSchemaObjectType = (schema) => immutableToJS(schema)?.type ?? "string"
```